### ISSUE TITLE
Column Block: Add More Styling

### ISSUE THREAD

**Issue by deborah86** [WordPress/gutenberg#30753]
## What problem does this address?

It is difficult to make the column block look nice without using additional CSS.

### Color options for a single column
There are no options to change the color of an entire column. The only option is to change the color of either the blocks within the column or the entire color itself.

![image](https://user-images.githubusercontent.com/12677420/114412425-f0432f80-9b61-11eb-8123-c058c4ef001c.png)

### Ability to Modify the Padding Between Column Elements

![image](https://user-images.githubusercontent.com/12677420/114412863-592aa780-9b62-11eb-8166-768558538607.png)

**Comment by paaljoachim** [comment]
Hi @deborah86 

I noticed you raised two concerns.
Color options for a single column

I just tested using WP 5.8 RC 2
Twenty Twenty One
The Gutenberg plugin version 11.0.0

This is what I see:

![Columns-block-adding-color](https://user-images.githubusercontent.com/5323259/125251948-224a6600-e2f8-11eb-8101-498f4359b8f7.jpg)

It looks like "Color options for a single column" has been dealt with.

---

The Ability to Modify the Padding Between Column Elements.
It looks like @andrewserong has been looking into this through this draft PR: https://github.com/WordPress/gutenberg/pull/32137

**Comment by andrewserong** [comment]
Yes, the color options for a single column task was addressed in: https://github.com/WordPress/gutenberg/pull/31778 and there's a follow-up fix for column background colors with vertical alignment that's under review in: https://github.com/WordPress/gutenberg/pull/32909

For adding in column gap support, we have a couple of steps before we can reliably get to it. Initially I explored adding in gap support directly to the columns block in #32137, however it turns out we need a more general approach for gap support that will be used across a number of blocks, so that we can better future-proof the feature. The next steps for gap support are tracked in #32366, but are broadly:

* Add a general gap block support feature (this is being worked on in #32571, but is currently waiting on its dependency #32392 landing first).
* Once the gap block support feature is in place, we can rework #32137 to use the gap block support feature, and use that PR to implement the feature that should close out this issue.

I'll continue updating #32366 as we make progress.
